src/dashboard/infrastructure/analytics-4/site-kit-analytics-4-api-call.php

```diff
@@ -0,0 +1,33 @@
+<?php
+// phpcs:disable Yoast.NamingConventions.NamespaceName.TooLong
+// phpcs:disable Yoast.NamingConventions.NamespaceName.MaxExceeded
+namespace Yoast\WP\SEO\Dashboard\Infrastructure\Analytics_4;
+
+use WP_REST_Request;
+use WP_REST_Response;
+
+/**
+ * Class that hold the code to do the REST call to the Site Kit api.
+ *
+ * @phpcs:disable Yoast.NamingConventions.ObjectNameDepth.MaxExceeded
+ */
+class Site_Kit_Analytics_4_Api_Call {
+
+	/**
+	 * The Analytics 4 API route path.
+	 */
+	private const ANALYTICS_DATA_REPORT_ROUTE = '/google-site-kit/v1/modules/analytics-4/data/report';
```

Comment:
Same question with the one I shared in https://github.com/Yoast/wordpress-seo/pull/22120:

Should we consider not using `/google-site-kit/v1` but rather `Google\Site_Kit\Core\REST_API\REST_Routes::REST_ROOT` instead

Reply:
I +1 on this